torch/distributed/tensor/_api.py

```diff
@@ -764,6 +764,51 @@ def distribute_tensor(
     )
 
 
+def _shard_tensor(
+    full_tensor: torch.Tensor,
+    placements: Sequence[Shard],
+    device_mesh: Optional[DeviceMesh] = None,
+) -> "DTensor":
+    """
+    Locally shards a full tensor based on indicated sharding arrangement, and
+    returns a DTensor containing the local shard.
+
+    .. warning:: This is a private API purposed to skip the communication
```

Comment:
Could we say `This is a private API that is subject to change. It is ...`

Reply:
Thanks, added now.